test/fixtures/basic-css-modules/src/index.js

```diff
@@ -0,0 +1,7 @@
+import twoStyles from './two.css';
```

Comment:
this is called `twoStyles` but contains only one definition. Was it intended to test if tree shaking works?

Reply:
there's no tree shaking for CSS, not sure why it was called `two`.

Reply:
ah ok, i thought it could be made. Checked it, even webpack does not remove unused css modules code (but they could because they know all referencing code) ^^